config/nunjucks/filters.js

```diff
@@ -144,15 +139,15 @@ function formatDateRangeAsRelativeWeek(value) {
   }
 
   if (_isCurrentWeek(dates)) {
-    return currentWeek
+    return 'this week'
```

Comment:
I think `i18n.t('actions::current_week'.toLowerCase())` would be better?

Reply:
I considered adding another translations file for time related stuff - the actions.json that is referencing is more specifically for labelling buttons and links. I'll raise a ticket to do that and go through and point the various areas of the service that  should use that at it, including the existing block further up in this file.